sdks/how-to/how-to-manage-healthcare-elements.md

```diff
@@ -4,3 +4,283 @@ description: Learn how to manage healthcare elements
+
+<!-- file://code-samples/how-to/manage-healthcare-elements/index.mts snippet:retrieve a HE as data owner-->
+```typescript
+const healthcareElementToRetrieve = new HealthcareElement({
+  description: 'To retrieve, I must create',
+})
+
+const createdHealthcareElement = await api.healthcareElementApi.createOrModifyHealthcareElement(
+  healthcareElementToRetrieve,
+  patient.id,
+)
+
+const retrievedHealthcareElement = await api.healthcareElementApi.getHealthcareElement(
+  createdHealthcareElement.id,
+)
+```
+
+:::caution
+
+Trying to retrieve a Healthcare Element you do not have access to will produce an error.
+
+:::
+
+## Modifying a Healthcare Element
+
+Given an existing Healthcare Element, it is possible to modify it.  
+Some fields, like id and rev, cannot be modified.
+
+<!-- file://code-samples/how-to/manage-healthcare-elements/index.mts snippet:modify a HE as data owner-->
+```typescript
+const yetAnotherHealthcareElement = await api.healthcareElementApi.createOrModifyHealthcareElement(
+  new HealthcareElement({
+    description: 'To modify, I must create',
+  }),
+  patient.id,
+)
+
+const modifiedHealthcareElement = {
+  ...yetAnotherHealthcareElement,
+  description: 'I can change and I can add',
+  openingDate: new Date('2019-10-12').getTime(),
+}
+
+const modificationResult = await api.healthcareElementApi.createOrModifyHealthcareElement(
+  modifiedHealthcareElement,
+  patient.id,
+)
+```
+
+If the operation is successful, the method returns the updated Healthcare Element as it is stored in the Backend.
+
+:::caution
+
+To update a Healthcare Element, both id and rev fields must be valid.
+
+:::
+
+## Retrieving Healthcare Element Using Complex Search Criteria
+
+If you want to retrieve a set of Healthcare Element that satisfy complex criteria, you can use a Filter.  
+In the following example, you will instantiate a filter to retrieve all the Healthcare Element of a Patient that a Healthcare Professional
+ can access
+
+<!-- file://code-samples/how-to/manage-healthcare-elements/index.mts snippet:create HE filter-->
+```typescript
+const healthcareElementFilter = await new HealthcareElementFilter()
+  .forDataOwner(user.healthcarePartyId)
+  .forPatients(api.cryptoApi, [patient])
+  .build()
+```
+
+:::note
+
+You can learn more about filters in the how to.
+
+:::
+
+After creating a filter, you can use it to retrieve the Healthcare Elements.
+
+<!-- file://code-samples/how-to/manage-healthcare-elements/index.mts snippet:use HE filter method-->
+```typescript
+const healthcareElementsFirstPage = await api.healthcareElementApi.filterHealthcareElement(
+  healthcareElementFilter,
+  undefined,
+  10,
+)
+```
+
+The `filter` method returns a PaginatedList that contains at most the number of elements stated
+ in the method's parameter. If you do not specify any number, the default value is 1000.  
+To retrieve more Healthcare Elements, you can call the same method again, using the startDocumentId provided in the previous
```

Comment:
To retrieve more Healthcare Elements, you can call the same method again, using the startDocumentId provided in the previous PaginatedList.